src/main/java/training/subsystems/elbow/IElbow.java

```diff
@@ -0,0 +1,15 @@
+package training.subsystems.elbow;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public interface IElbow {
+
+    Rotation2d getPosition();
+
+    Rotation2d getVelocity();
```

Comment:
the IElbow should not have any functions that return anything, for getting positions and velocities use Inputs (in this case ask me how to do that)